packages/xo-server-test/src/_xoConnection.js

```diff
@@ -128,6 +128,16 @@ class XoConnection extends Xo {
     })
   }
 
+  async createTempNetwork(params) {
+    const id = await this.call('network.create', {
+      name: 'XO Test',
+      pool: config.pools.default,
+      ...params,
+    })
+    this._tempResourceDisposers.push('network.delete', { id })
+    return this.getOrWaitObject(id)
+  }
+
```

Comment:
Keep method sorted.

Reply:
Should i keep method sorted by context or by name? Did this comment related to this change or to the existent sort?

Reply:
Only to this change, this method should be at least above `createTempVm`.